app/src/main/java/nodomain/freeyourgadget/gadgetbridge/model/NotificationType.java

```diff
@@ -35,7 +35,7 @@
     GENERIC_EMAIL(PebbleIconID.GENERIC_EMAIL, PebbleColor.Orange),
     GENERIC_NAVIGATION(PebbleIconID.LOCATION, PebbleColor.Orange),
     GENERIC_PHONE(PebbleIconID.DURING_PHONE_CALL, PebbleColor.JaegerGreen),
-    GENERIC_SMS(PebbleIconID.GENERIC_SMS, PebbleColor.VividViolet),
+    GENERIC_SMS(PebbleIconID.GENERIC_SMS, PebbleColor.IslamicGreen),
```

Comment:
Why the change for SMS?

Reply:
Official App uses `IslamicGreen` for messages, not `VividViolet`.

Reply:
Effectively what @aveao said.

I noted through testing that a value was unexpectedly violet, so I changed the SMS field. This is a follow-up of sorts to #816.